report/Evaluation/Discussion.tex

```diff
@@ -0,0 +1,26 @@
+\section{Discussion}
+
+Throughout this chapter we will evaluate the solution to the challenge that this project aimed to solve.
+
+\subsection{Solution to The Challenge}
+
+The Challenge that this project aimed to solve was as described in~\autoref{subsec:theChallenge}:
+
+\begin{quote}
+    Improve the communication between customers and developers in an Agile Software Development Process.
+\end{quote}
+
+The solution developed during this project, which is described from~\autoref{cha:sprintOne} to~\autoref{cha:sprint_4} was supposed to improve the communication between customers and developers.
+However during the project period, the world was hit by the Corona Virus and to verify that the developed solution actually solves this problem, the product should be tested on development teams and customers.
+
+However, since the project team, was both acting as both costumer and developer, the team itself is a valid organization to validate the solution.
```

Comment:
```suggestion
Since the project group, was both acting as customer and developer, the group itself is a valid organization to validate the solution.
```